sentry_core.go

```diff
@@ -44,6 +64,13 @@ func EventLevel(level zapcore.Level) SentryCoreOption {
 	}
 }
 
+// TagMap will set map to match zap fields with sentry tags.
+func TagMap(tagMap FieldTagMap) SentryCoreOption {
+	return func(w *SentryCore) {
+		w.TagMap = tagMap
+	}
+}
+
```

Comment:
I think `UserTags` and `GenericTags` should be separate options. It makes sense to combine them from implementation point of view but for end user of library this is a different things and some of them wants only one of them